test/functional/ex_cmds/mkview_spec.lua

```diff
@@ -0,0 +1,74 @@
+local lfs = require('lfs')
+local helpers = require('test.functional.helpers')(after_each)
+
+local clear = helpers.clear
+local command = helpers.command
+local get_pathsep = helpers.get_pathsep
+local eq = helpers.eq
+local funcs = helpers.funcs
+
+local file_prefix = 'Xtest-functional-ex_cmds-mkview_spec'
+
+describe(':mkview', function()
+  local tmp_file_base = file_prefix .. '-tmpfile'
+  local local_dir = file_prefix .. '.d'
+  local view_dir = file_prefix .. '.view.d'
+
+  before_each(function()
+    clear()
+    lfs.mkdir(view_dir)
+    lfs.mkdir(local_dir)
+  end)
+
+  after_each(function()
+    -- Remove any views created in the view directory
+    full_view_dir_path = lfs.currentdir() .. get_pathsep() .. view_dir
+    for file in lfs.dir(view_dir) do
+      full_file_path = full_view_dir_path .. get_pathsep() .. file
+      if lfs.attributes(full_file_path, "mode") == "file" then
+        os.remove(full_file_path)
+      end
+    end
+    lfs.rmdir(view_dir)
+    lfs.rmdir(local_dir)
```

Comment:
`helpers.rmdir()` should be more robust / avoids the need for lines 26-31.

Reply:
Fixed